Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java

```diff
@@ -39,18 +40,19 @@
     public void onCreate() {
         super.onCreate();
         //init MoPub SDK
-        List<String> networksToInit = new ArrayList<String>();
-        networksToInit.add("com.mopub.mobileads.VungleRewardedVideo");
         SdkConfiguration sdkConfiguration = new SdkConfiguration.Builder("a935eac11acd416f92640411234fbba6")
-                .withNetworksToInit(networksToInit)
                 .build();
         MoPub.initializeSdk(this, sdkConfiguration, null);
         //set Prebid Mobile global Settings
         //region PrebidMobile API
-        PrebidMobile.setPrebidServerAccountId(Constants.PBS_ACCOUNT_ID);
-        PrebidMobile.setPrebidServerHost(Host.APPNEXUS);
+        PrebidMobile.setPrebidServerAccountId("account");
+        Host custom = Host.CUSTOM;
+        custom.setHostUrl("http://10.0.2.2:8000/openrtb2/auction");
```

Comment:
Is it necessary to use a custom address ?

Reply:
This is for testing. Will change it back after end to end testing